test/e2e/nodeportlocal_test.go

```diff
@@ -137,7 +138,16 @@ func checkForNPLRuleInIPTables(t *testing.T, data *TestData, r *require.Assertio
 func deleteNPLRuleFromIPTables(t *testing.T, data *TestData, r *require.Assertions, antreaPod string, rule nplRuleData) {
 	cmd := append([]string{"iptables", "-w", "10", "-t", "nat", "-D", "ANTREA-NODE-PORT-LOCAL"}, buildRuleForPod(rule)...)
 	t.Logf("Deleting iptables rule for %v", rule)
-	_, _, err := data.runCommandFromPod(antreaNamespace, antreaPod, agentContainerName, cmd)
+	const timeout = 30 * time.Second
+	err := wait.Poll(time.Second, timeout, func() (bool, error) {
+		_, _, err := data.runCommandFromPod(antreaNamespace, antreaPod, agentContainerName, cmd)
+		if err != nil {
+			t.Logf("Error while deleting rule from iptables: %v", err)
+			// Retry, as sometimes error can occur due to concurrent operations on iptables.
```

Comment:
I don't think we need this, I added `-w 10` to the iptables delete command already to accommodate for this issue

Reply:
Removed

Reply:
Removed